result/yoda.js

```diff
@@ -0,0 +1,18 @@
+
+/*eslint no-multi-spaces: "error"*/
+
+var a =  1;
+var a =  1;
+var a =  1;
+
+if(foo   === "bar") {}
+
+a <<  b
+a <<  b
+
+var arr = [1,  2];
+
+a ?  b: c
```

Comment:
<!-- Comment by [SideCI](https://sideci.com) -->
__[development][eslint]__
* Error - Multiple spaces found before 'b'. (no-multi-spaces)
* Error - 'b' is not defined. (no-undef)
* Error - 'c' is not defined. (no-undef)